lumigator/python/mzai/backend/backend/tests/api/routes/test_api_workflows.py

```diff
@@ -0,0 +1,51 @@
+from time import sleep
+
+import pytest
+import requests
+from fastapi.testclient import TestClient
+from schemas.datasets import DatasetFormat, DatasetResponse
+
+
+# First test waits and polls up to 10 times for the real backend to be up.
+def test_health_ok(local_client: TestClient):
+    attempt = 0
+    while attempt < 10:
```

Comment:
```suggestion
    while attempt < range(10)
```
then we don't need to increment a counter

Reply:
Right you are 😆 
I'll touch that next PR I open if someone doesn't fix that sooner.